src/core/tests/type_prop/group_normalization.cpp

```diff
@@ -0,0 +1,193 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_test_utils/test_assertions.hpp"
+#include "gtest/gtest.h"
+#include "openvino/openvino.hpp"
+#include "openvino/opsets/opset12.hpp"
+#include "util/type_prop.hpp"
+
+using namespace ov;
+using namespace testing;
+
+TEST(type_prop, group_normalization_basic) {
+    const auto data = std::make_shared<opset12::Parameter>(element::f32, Shape{1, 12, 6, 6});
+    const auto scale = std::make_shared<opset12::Parameter>(element::f32, Shape{12});
+    const auto bias = std::make_shared<opset12::Parameter>(element::f32, Shape{12});
+
+    const auto gn = std::make_shared<opset12::GroupNormalization>(data, scale, bias, 4, 0.00001f);
+    EXPECT_EQ(gn->get_element_type(), element::f32);
+    EXPECT_EQ(gn->get_shape(), (Shape{1, 12, 6, 6}));
+}
+
+TEST(type_prop, group_normalization_labels) {
+    auto data_shape = PartialShape{1, 12, 6, 6};
```

Comment:
Set also labels on other inputs to check that others not propagated (e.g. when some dimension are merged etc).